style.css

```diff
@@ -1,111 +1,788 @@
-  padding: 25px 20px;
-  text-align: center;
-  font-size: 40px;
-  font-weight: 500;
-}
-.middle_section p span {
-  font-size: 72px;
-  font-weight: 700;
-}
-
-/* card section  */
-
-.card_section {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: center;
-  padding: 70px 100px;
-  gap: 40px;
-}
-.card_section-card {
-  width: auto;
-  box-shadow: 3px 5px 5px rgba(1, 1, 1, 0.2);
-  border-radius: 10px;
-}
-.card_section-card img {
-  width: 300px;
-  height: 200px;
-}
-.card_section-text {
-  max-width: 300px;
-  text-align: start;
-}
-.card_section-text p {
-  display: flex;
-  flex-direction: column;
-  text-align: justify;
-  padding: 5px 15px;
-}
-.card_section-heading {
-  font-size: 20px;
-  font-weight: 500;
-}
-
-/* last section  */
-
-.last_section {
-  display: flex;
-  flex-direction: row;
-}
-.last_section-contact {
-  background-color: #0369b8;
-  color: white;
-  padding: 10px 25px;
-}
-.last_section-heading1 {
-  font-size: 25px;
-  font-weight: 600;
-}
-.last_section-about {
-  background-color: #b9098d;
-  color: white;
-  padding: 10px 25px;
-}
-.last_section-heading2 {
-  font-size: 25px;
-  font-weight: 600;
-}
-.footer {
-  background-color: black;
-  padding: 15px 25px;
-  color: white;
-  text-align: center;
-  font-weight: 500;
-  font-size: 15px;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: sans-serif;
 }
+
+html {
+    scroll-behavior: smooth;
+
+}
+
+section {
+    width: 100%;
+    height: 100vh;
+}
+
+section nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    position: fixed;
+    right: 0;
+    left: 0;
+    background: white;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+    z-index: 1000;
+}
+
+section nav .logo img {
+    width: 100px;
+    cursor: pointer;
+    margin: 7px 0px;
+}
+
+section nav ul {
+    list-style: none;
+
+}
+
+section nav ul li {
+    display: inline-block;
+    margin: 0 15px;
+}
+
+section nav ul li a {
+    text-decoration: none;
+    color: black;
+    font-weight: 500;
+    font-size: 17px;
+}
+
+section nav ul li a::after {
+    content: '';
+    width: 0%;
+    height: 2px;
+    background: #fac031;
+    display: block;
+    transition: 0.2s linear;
+}
+
+section nav ul li a:hover:after {
+    width: 100%;
+}
+
+section nav ul li a:hover {
+    color: #fac031;
+
+}
+
+section nav .icon i {
+    font-size: 18px;
+    color: #000;
+    margin: 0 5px;
+    cursor: pointer;
+    transition: 0.3s;
+}
+
+section nav .icon i:hover {
+    color: #fac031;
+}
+
+section .main {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    position: relative;
+    top: 130px;
+}
+
+section .main .main_text h1 {
+    font-size: 50px;
+    position: relative;
+    top: -90px;
+    left: 22px;
+
+}
+
+section .main .main_text h1 span {
+    margin-left: 15px;
+    color: #fac031;
+    font-family: cursive;
+    line-height: 20px;
+    font-size: 70px;

```

Comment:
use relative values such as rem and em to keep the font responsive as per content